api/apps/api/src/modules/scenarios/input-parameter-file.provider.ts

```diff
@@ -84,12 +84,20 @@ export class InputParameterFileProvider {
   ) {}
 
   async getInputParameterFile(scenarioId: string): Promise<string> {
-    const scenario = await this.scenariosService.getById(scenarioId);
+    const scenario = await this.scenariosService.getById(scenarioId, {
+      include: ['project', 'project.organization'],
+    });
     const inputParameterFile = new InputParameterFile(
       this.ioSettings,
       scenario.boundaryLengthModifier,
       scenario.numberOfRuns,
-      scenario.metadata?.marxanInputParameterFile,
+      {
+        ...scenario.metadata?.marxanInputParameterFile,
+        _CLOUD_SCENARIO: scenario.name,
+        _CLOUD_PROJECT: scenario.project?.name ?? 'NA',
+        _CLOUD_ORGANIZATION: scenario.project?.organization?.name ?? 'NA',
+        _CLOUD_GENERATED_AT: new Date().toISOString(),
```

Comment:
in case this got lost in the noise of unrelated failing tests, the snapshot that includes these will need updating (see CI output for `scenario-input-files/scenario-input-files.e2e-spec.ts`)

Reply:
@hotzevzl haven't finished yet, its a draft ; )

yep, thats the only thing left before being ready to review, thanks tho for taking a look at it! ❤️